### ISSUE TITLE
Avocode 2.2.1 Mac 10.10.3 - Your Avocode has gone bad

### ISSUE THREAD

**Issue by hadabo** [avocode/avocode#327]
I left my laptop for half an hour, and when i came back i found more than 6 notifications from Avocode!

-----

```
TypeError: object is not a function
  at t.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:26467)
  at t.EventEmitter.emit (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/node_modules/eventemitter2/lib/eventemitter2.js:339:22)
  at e.setOfflineState (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/user.js:1:2886)
  at e._handleOfflineState (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/user.js:1:3745)
  at e.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/user.js:1:342)
  at emitNone (events.js:72:20)
  at emit (events.js:163:7)
  at e._handleOfflineState (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/api.js:1:9271)
  at e._handleOfflineState (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/api.js:1:202)
  at e.setOfflineState (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/api.js:1:1791)
  at t.suspend (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:22239)
  at HTMLBodyElement.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:950)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Avocode.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at CommandRegistry.handleCommandEvent (/Applications/Avocode.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Avocode.app/Contents/Resources/app.asar/src/co
```

**Comment by hadabo** [comment]
![screen shot 2015-07-01 at 14 46 32](https://cloud.githubusercontent.com/assets/172296/8453807/ccb8ceae-2000-11e5-86fa-9a394a70c714.png)

**Comment by rychlis** [comment]
Hi, this should be fixed in 2.3.0, I'm closing the issue.
